attest/core/src/nonce.rs

```diff
@@ -139,15 +138,18 @@ impl FromHex for IasNonce {
             return Err(EncodingError::InvalidInputLength);
         }
         let mut retval = Self::default();
-        hex2bin(s.as_bytes(), &mut retval.0[..])?;
+        hex::decode_to_slice(s, &mut retval.0[..])?;
         Ok(retval)
     }
 }
 
 impl ToHex for IasNonce {
     fn to_hex(&self, dest: &mut [u8]) -> Result<usize, usize> {
-        match bin2hex(self.as_ref(), dest) {
-            Ok(buffer) => Ok(buffer.len()),
+        let bytes = self.as_ref();
+        match hex::encode_to_slice(bytes, dest) {
+            // Previously binascii was used instead of hex. 
+            // Binascii would always return input.len() * 2.
+            Ok(()) => Ok(bytes.len()*2),
```

Comment:
Is it still needed to return input.len() * 2

Reply:
Yes, the `ToHex` trait requires it

Reply:
Helpful to know!